aptos-move/framework/aptos-framework/sources/transaction_context.move

```diff
@@ -1,4 +1,29 @@
 module aptos_framework::transaction_context {
+
+    /// A wrapper denoting universally unique identifer (UUID)
+    /// for storing an address
+    struct UUID has drop, store {
+        unique_address: address
+    }
+
+    /// Return the transaction hash of the current transaction
+    public native fun get_txn_hash(): vector<u8>;
+
+    /// Return a universally unique identifier (of type address) generated
+    /// by hashing the transaction hash of this transaction and a sequence number 
+    /// specific to this transaction. This function can be called any
+    /// number of times inside a single transaction. Each such call increments
+    /// the sequence number and generates a new unique address
+    public native fun create_unique_address(): address;
```

Comment:
Add to the comment:

- uses Scheme in types/src/transaction/authenticator.rs for domain separation from other ways of generating unique addresses.

Reply:
Done.